instrumentation/openai-client-instrumentation/instrumentation-0.13.0/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+  alias(catalog.plugins.muzzleGeneration)
+  alias(catalog.plugins.muzzleCheck)
+  id("elastic-otel.instrumentation-conventions")
+}
+
+val openAiVersion = "0.13.0"; // DO NOT UPGRADE
```

Comment:
Is it something that renovate/dependabot would attempt to upgrade ? If so I think there are ways like splitting the string in parts to prevent this.

Reply:
Good idea. I don't know if renovate is smart enough for this. We can just keep it like this for now and see if renovate detects it and attempts an upgrade